docs/Systems.md

```diff
@@ -220,35 +221,84 @@ cd rpi2_out
+
+Once you're happy with your System, you can package it by changing to the `rpi2_out` directory and running:
+
+```
+make system
+```
+
+This will create a `<system>.tar.gz` file that can be hosted on a web server and referenced from a Hex package just like the official Nerves Systems are.
+
+### Supporting New Target Hardware
+
+If you're trying to support a new Target, there may be quite a bit more work involved, depending on how mature the support for that hardware is in the Buildroot community.
+If you're not familiar with [Buildroot](https://buildroot.org/), you should learn about that first, using the excellent training materials on their website.
+
+If you can find an existing Buildroot configuration for your intended hardware and you want to get it working with Nerves:
 
-### Additional package configuration
+1.  Follow their procedure and confirm your target boots (independent of Nerves).
 
-If you have used Buildroot before, the workflow is similar. The most common task
-will be enabling C/C++ packages for Nerves or tweaking the Linux kernel. Here is
-a short overview of commands:
+2.  Figure out how to get everything working with the version of Buildroot Nerves uses. See [the `NERVES_BR_VERSION` variable in `create-build.sh`](https://github.com/nerves-project/nerves_system_br/blob/master/create-build.sh).
 
-  1. Select packages by running `make menuconfig`
-  2. Modify the Linux kernel with `make linux-menuconfig`
-  3. Enable more command line utilities, `make busybox-menuconfig`
+  * Look for packages and board configs can need to be copied into your System.
+  * Look for patches to existing packages that are needed.
 
-The changes are stored temporarily. To save them back in your system image, run
-one or more of the following:
+3. Create a defconfig that mimics the one from step 1, and get `nerves_system_br` to build it.
 
-  1. `make savedefconfig` updates the `nerves_defconfig` in your system
-     repository. It contains the Buildroot configuration
-  2. Run `make linux-savedefconfig` and `cp build/linux-x.y.z/defconfig <your
-     system>` to save the Linux configuration. If your system doesn't contain a
-     custom Linux configuration yet, you'll need to update the Buildroot
-     configuration to point to the new Linux defconfig in your system directory.
-     The path is usually something like `$(NERVES_DEFCONFIG_DIR)/linux-x.y_defconfig`
-  3. For Busybox, the convention is to copy `build/busybox-x.y.z/.config` to a
-     file in the system repository. Like the Linux configuration, the Buildroot
-     configuration will need to be updated to point to the custom config.
+> NOTE: You probably want to disable any userland packages that may be included by default to avoid distraction.
 
-The Buildroot [user manual](http://nightly.buildroot.org/manual.html) can be
```

Comment:
Is the Buildroot documentation linked elsewhere? Given how often I refer to it, it seems important to me to include it somewhere, but I could be talked out of it.